### ISSUE TITLE
component lists rendered with v-for should have explicit keys.

### ISSUE THREAD

**Issue by vinicius73** [quasarframework/quasar#439]
Vue 2.2 have a deprecation
https://github.com/vuejs/vue/releases/tag/v2.2.0
https://vuejs.org/v2/guide/list.html#key

```
<q-list-item v-for="opt in options">: component lists rendered with v-for should have explicit keys. See https://vuejs.org/guide/list.html#key for more info.
```

I have not had time to send a PR yet, I am leaving this issue to register the problem, and if any people want to correct it can be at ease.

**Comment by rstoenescu** [comment]
This is already fixed for v0.14. There are lots of places with "v-for" which should be added a key for v0.13. Marking this as contributor friendly. Hoping my fever would go down soon so I can resume work, so anyone can jump with a PR on this one to speed up things.

**Comment by rstoenescu** [comment]
Done for edge and future v0.13.2 too.
